src/Microsoft.SqlTools.ServiceLayer/Migration/Contracts/MigrationObjectProperties.cs

```diff
@@ -0,0 +1,129 @@
+using System.Collections.Generic;
+using Microsoft.SqlServer.Migration.Assessment.Common.Contracts.TargetAssessment.Models;
+
+namespace Microsoft.SqlTools.ServiceLayer.Migration.Contracts
+{
+    public class ServerProperties
```

Comment:
ServerProperties is a bit generic - even if it's in the Migration namespace I'd suggest naming it something more clear like `ServerAssessmentProperties`. (especially since it contains migration/assessment specific properties - not just properties of the server itself)